improver_tests/utilities/test_DifferenceBetweenAdjacentGridSquares.py

```diff
@@ -231,7 +190,9 @@ def test_3d_cube(self):
         )
         expected_x = np.array([[[1, 1], [2, 2], [5, 5]], [[1, 1], [0, 4], [5, 10]]])
         expected_y = np.array([[[1, 2, 3], [3, 6, 9]], [[1, 0, 3], [3, 8, 14]]])
-        cube = set_up_cube(data, "wind_speed", "m s-1", realizations=np.array([1, 2]))
+        cube = set_up_variable_cube(
+            data, "wind_speed", "m s-1", "equalarea", realizations=np.array([1, 2]),
+        )
```

Comment:
Almost all the tests in this set are working with integer data. As such we should just use ArrayEquals rather than AlmostEquals. Tests on the following lines can all be modified to use ArrayEquals (all the ones comparing the data result that are not float32):
- 66, 78, 108, 117, 164, 166, 198, 200